src/Option.ts

```diff
@@ -70,6 +70,18 @@ export class None<A> {
   reduce<B>(b: B, f: (b: B, a: A) => B): B {
     return b
   }
+  /**
+   * `alt` short for alternative, takes another `Option`. If this `Option` is a `Some` type
+   * then it will be returned, if it is a `None` then it will return the next `Some` if it exist. If
+   * both are `None` then it will return `none`.
+   *
+   * For example
+   * `const someFn = (o: option.Option<number>) => o.alt(option.some(4))`
```

Comment:
@piq9117 I think we can omit `option.`. `Option`, `some` and `none` should be clear in this context. Also do not include the generated docs (I'll handle that before releasing next version, thanks)

Reply:
gotcha. thanks!

Reply:
@piq9117 thanks, last thing before merging

> do not include the generated docs (I'll handle that before releasing next version, thanks)

Reply:
totally forgot about that. Hopefully I didn't miss anything. 😅

Reply:
@piq9117 sorry I wasn't clear, I meant to revert the changes to docs/api/md/Option.md, not to remove the file

Reply:
@piq9117 ok never mind, I'll handle this in the next commits. Thanks for this PR